src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator-wrapper">
+      <div className="row1 Display">0</div>
+      <div className="row2">
+        <button type="button" className="calculator-button">AC</button>
+        <button type="button" className="calculator-button">+/-</button>
+        <button type="button" className="calculator-button">%</button>
+        <button type="button" className="calculator-button calculator-operator">&#247;</button>
```

Comment:
- [ ] It would be better if we create a child component for calculator.js component. We can create a `Button` component 
![Screenshot from 2023-09-22 02-06-14](https://github.com/efmomoh/math-magician/assets/46388113/10725653-dd08-4696-8bdf-3143df2fceab)